### ISSUE TITLE
Torrentz2 not working - 503: Service Unavailable

### ISSUE THREAD

**Issue by JonathanGorbutt** [pymedusa/SickRage#1053]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:Master
OS:Windows Server 2008 R2
What you did: Searched for episodes manually to with only Torrentz2 provider enabled
What happened: No episodes found
What you expected: Found Episodes
Logs:

```
2016-09-08 19:41:31 INFO     SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Unable to find manual results for: [The Big Bang Theory - S09E24 - The Convergence Convergence]
2016-09-08 19:41:31 INFO     SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Performing season pack search for The Big Bang Theory
2016-09-08 19:41:29 INFO     SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Performing episode search for The Big Bang Theory
2016-09-08 19:41:29 INFO     SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Using manual search providers
2016-09-08 19:41:29 INFO     SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Beginning manual search for: [The Big Bang Theory - S09E24 - The Convergence Convergence]
2016-09-08 19:31:27 INFO     SEARCHQUEUE-DAILY-SEARCH :: [9c6346b] No needed episodes found
2016-09-08 19:31:26 INFO     SEARCHQUEUE-DAILY-SEARCH :: [9c6346b] Beginning daily search for new episodes
```

**Comment by JonathanGorbutt** [comment]
Debug Log:

016-09-08 19:47:37 INFO     SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Unable to find manual results for: [The Big Bang Theory - S09E24 - The Convergence Convergence]
2016-09-08 19:47:37 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [9c6346b] 80379: Formatting pattern: %SN - S%0SE%0E - %EN -> The Big Bang Theory - S09E24 - The Convergence Convergence
2016-09-08 19:47:37 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Parsed D:\Jonny's Stuff\TVShows\The Big Bang Theory\Season 9\The Big Bang Theory - 9x24 - The Convergence Convergence into title: The Big Bang Theory, season: 9, episode: [24], episode_title: The Convergence Convergence, type: episode, absolute_episode: [], quality: Unknown
2016-09-08 19:47:37 INFO     SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Performing season pack search for The Big Bang Theory
2016-09-08 19:47:37 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Fallback season pack search initiate
2016-09-08 19:47:37 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] No data returned from provider
2016-09-08 19:47:37 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Requested url https://torrentz2.eu/feed?f=Big+Bang+Theory+S09E24 returned status code 503: Service Unavailable
2016-09-08 19:47:37 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] GET URL: https://torrentz2.eu/feed?f=Big+Bang+Theory+S09E24 [Status: 503]
2016-09-08 19:47:36 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Search string: Big Bang Theory S09E24
2016-09-08 19:47:36 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] No data returned from provider
2016-09-08 19:47:36 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Requested url https://torrentz2.eu/feed?f=The+Big+Bang+Theory+S09E24 returned status code 503: Service Unavailable
2016-09-08 19:47:36 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] GET URL: https://torrentz2.eu/feed?f=The+Big+Bang+Theory+S09E24 [Status: 503]
2016-09-08 19:47:35 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Search string: The Big Bang Theory S09E24
2016-09-08 19:47:35 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] No data returned from provider
2016-09-08 19:47:35 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Requested url https://torrentz2.eu/feed?f=TBBT+S09E24 returned status code 503: Service Unavailable
2016-09-08 19:47:35 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] GET URL: https://torrentz2.eu/feed?f=TBBT+S09E24 [Status: 503]
2016-09-08 19:47:34 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Search string: TBBT S09E24
2016-09-08 19:47:34 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Search mode: Episode
2016-09-08 19:47:34 INFO     SEARCHQUEUE-MANUAL-80379 :: [Torrentz2] :: [9c6346b] Performing episode search for The Big Bang Theory
2016-09-08 19:47:34 INFO     SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Using manual search providers
2016-09-08 19:47:34 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Internal name cache for The Big Bang Theory set to: [ tbbt, big bang theory, the big bang theory ]
2016-09-08 19:47:34 INFO     SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Beginning manual search for: [The Big Bang Theory - S09E24 - The Convergence Convergence]
2016-09-08 19:47:34 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [9c6346b] 80379: Formatting pattern: %SN - S%0SE%0E - %EN -> The Big Bang Theory - S09E24 - The Convergence Convergence
2016-09-08 19:47:34 DEBUG    SEARCHQUEUE-MANUAL-80379 :: [9c6346b] Parsed D:\Jonny's Stuff\TVShows\The Big Bang Theory\Season 9\The Big Bang Theory - 9x24 - The Convergence Convergence into title: The Big Bang Theory, season: 9, episode: [24], episode_title: The Convergence Convergence, type: episode, absolute_episode: [], quality: Unknown
2016-09-08 19:47:14 INFO     Thread-27 :: [9c6346b] Broken providers found: [u'btdigg', u'kickasstorrents', u'torrentz']

**Comment by medariox** [comment]
From time to time Torrentz2 uses CloudFlare's bot protection and this doesn't allow Medusa to do it's scraping. We have a PR that is going to fix this, once it's merged: https://github.com/pymedusa/SickRage/pull/1014

**Comment by ratoaq2** [comment]
Fixed by #1014
